mousemapper.sh

```diff
@@ -4,16 +4,17 @@ event_type=EV_KEY
 action_type=POINTER_BUTTON
 pressed="pressed,"
 
-readarray -t devices <<<$(libinput list-devices | grep pointer -B3 | grep -o '/dev/input/event[1-9]*')
+readarray -t devices <<<$(libinput list-devices | grep pointer -B3 | grep -o '/dev/input/event[0-9]*')
+keyboard=$(ls /dev/input/by-path/*-kbd)
```

Comment:
Won't this break if you have two keyboards?

Reply:
fixed